### ISSUE TITLE
refactor: Install Helm Controller via our own Helm chart

### ISSUE THREAD

**Issue by axisofentropy** [UffizziCloud/uffizzi-cluster-operator#15]
As of this writing, this operator is installing `flux` by itself by calling its CLI. https://github.com/UffizziCloud/uffizzi-cluster-operator/blob/4adf432d0d9a0eae98f1a8ee3f42a0ba41664a49/controllers/uffizzicluster_controller.go#L401

These two flux components should probably be installed as dependencies to our Helm chart.

This operator probably should not have sufficient permissions to install this itself.

This is a low priority but will need to happen before we lock down this operator's RBAC. It's possible we replace flux with a different Helm controller.

**Comment by waveywaves** [comment]
completed, helm controller and source controller are installed as deps from the operator's help chart
